src/ConnectionManager.js

```diff
@@ -114,9 +116,18 @@ define(["structures/Response", "structures/Request", "structures/CAPIError"],
             headers['X-Siteaccess'] = this._siteAccess;
         }
 
+        requestUrl = this._endPointUrl + url;
+
+        if (this._siteAccess && this._token) {
+            headers['X-Siteaccess'] = this._siteAccess;
+            headers['X-CSRF-Token'] = this._token;
+
+            requestUrl = url;
```

Comment:
If you don't need `this._endPointUrl` then we should rather pass an empty string from outside of this function.